.github/workflows/python-publish.yml

```diff
@@ -6,17 +6,29 @@ name: "Upload Python Package to PyPi"
 on:
   release:
     types: ["published"]
+
 jobs:
-  build:
-    runs-on: "ubuntu-latest"
+  pypi-publish:
```

Comment:
Have you already created the PyPi account? Will this also be accessible to all owners of the Pytoyoda Github organisation?

Reply:
The organization request seems to be stuck in a limbo on pypi. It seems that I have to first upload the package to give out further permissions.